### ISSUE TITLE
Distributing superdirt

### ISSUE THREAD

**Issue by yaxu** [musikinformatik/SuperDirt#117]
Installing tidal and superdirt is difficult, it'd be great to have something that can be downloaded and ran..

What are the issues involved with providing a download with supercollider, the required sc3-plugins and the superdirt quark (plus dependencies)? With downloads for windows, mac and linux? Ideally also with tidal binary bindled but that's a separate issue..

**Comment by telephon** [comment]
What is the most difficult part of the install process?

**Comment by yaxu** [comment]
I generally only hear of cases that go badly, not successes, but often see the following problems:

- Linux distribution packages are generally broken, requiring a compile of sc3 and sc3-plugins
- git is required for quarks, and if it's missing failures are pretty silent
- dirt-samples takes a long time to install with no indication that it's still running, and (I think) no warning if you close supercollider during the process.. then you get a situation that requires advanced git/filesystem knowledge recover from (generally the quarks system seems be error prone and difficult to recover from errors)
- sc3-plugins is still difficult to install (it would be great if the stuff that's good enough for superdirt could go in supercollider core..)

More generally, a tidal install requires a few different components, and install processes aimed at developers, each of which assumes domain knowledge, and each of which depends on a community of actors not to make mistakes and set of servers to be online and working.. I think we need a single download-and-click-to-run, with editor, tidal interpreter and sound system bundled and tested as working together.

**Comment by telephon** [comment]
Apart from the first issue, at least for macOS, all the rest could be solved by distributing a standalone. I'm not sure if I'll have the time to maintain the distribution, maybe someone could volunteer in doing this.

The first is an issue that should be submitted to the sc3-plugins project.
The second should be submitted to supercollider, it should definitely warn.

**Comment by yaxu** [comment]
With some ongoing process in making a relocatable binary for Tidal, it would be good to return to this. I see there are [instructions for macOS](https://depts.washington.edu/dxscdoc/Help/Guides/standalones.html) in making a 'standalone', but this process isn't the same for linux/windows. 

This seems to be the current state of play: https://github.com/supercollider/supercollider/issues/4353

There's [something here for linux](https://github.com/miguel-negrao/scStandalone), which depends on a four-year old branch of supercollider to create 'projects'.

**Comment by telephon** [comment]
That might be a good question on http://scsynth.org/

**Comment by yaxu** [comment]
Yes helpful responses https://scsynth.org/t/deploying-supercollider-sclang-standalone-apps/4030

**Comment by cdbzb** [comment]
this PR (awaiting review)  [ #6068 ](https://github.com/supercollider/supercollider/pull/6068) adds support for relative paths in sclang_conf.yaml as well as supporting multiple config files
